trading_funcs.py

```diff
@@ -90,9 +112,16 @@ def process_earnings(dates: List, diffs: List, start_date: str, end_date: str):
 
 def process_flips(ema12: np.array, ema26: np.array) -> List:
     """
-    Returns list of when 12-day ema and 26-day ema flip
-    
-    mostly used for MACD
+    The purpose of this function is to process the flips between the 12-day ema and 26-day ema. It
+    is primarily used for the MACD indicator. This is done by iterating through the ema12 and ema26
+    and appending the True or False value to the temp list. 
+
+    Args:
+        ema12 (np.array): The 12-day ema which is used to get the flips
+        ema26 (np.array): The 26-day ema which is used to get the flips
+
+    Returns:
+        List[map(int, temp)]: The list of flips between the 12-day ema and 26-day ema
```

Comment:
It is a list of ints. The last line basically makes it into a list of ints. I changed in my `Bot` branch so it appends ints instead of bools any ways.

Put something like
List: description. List of ints, etc